modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -5,20 +5,26 @@
 [id="installation-osp-kuryr-known-limitations_{context}"]
 = Known limitations of installing with Kuryr
 
-There are known limitations when using Kuryr SDN:
+There are known limitations when using Kuryr SDN, depending on the
+{rh-openstack} version:
 
 * An Amphora load balancer VM is deployed per OpenShift Service with
 the default Octavia load balancer driver (Amphora driver). If the environment is
-resource constrained, creating a large amount of Services could be a problem.
+resource constrained, creating a large amount of Services could be a problem as
+one VM is created for each Service.
 * Depending on the Octavia version, UDP listeners are not supported. This means
-that OpenShift UDP Services are not supported.
-* There is a known limitation of Octavia not supporting listeners on different
-protocols, like UDP and TCP, on the same port. Thus, Services exposing the same
-port for different protocols are not supported.
+that OpenShift UDP Services are not supported. UDP listeners are supported from
+{rh-openstack} 16+.
+* Depending on the Octavia version, there is a known limitation of Octavia
```

Comment:
Be explicit about the version in this line.

Reply:
"Octavia version X cannot listen to multiple protocols, like UDP and TCP, on the same port. Service that expose the same port to different services are not supported. Listeners that use different protocols are supported in {rh-openstack} version 16 and later."